resources/solr-configuration/src/main/resources/patients/conf/schema.xml

```diff
@@ -62,11 +62,10 @@
     <field name="visibility" type="string" indexed="true" stored="true" required="false" />
     <field name="accessLevel" type="int" indexed="true" stored="true" required="false" />
 
-    <!-- Index all fields ending in "phenotype", only store those that start with "actual_" -->
-    <dynamicField name="actual_*" type="text_ws" indexed="true" stored="true" multiValued="true"/>
-    <!-- We don't use the full "phenotype" in the pattern since Solr applies the longest matching pattern,
-         so we must keep it shorter than "actual_" -->
-    <dynamicField name="*notype" type="text_ws" indexed="true" stored="false" multiValued="true"/>
+    <!-- Index all fields ending in "phenotype", not storing those that start with "extended_" -->
+    <!-- Solr applies the longest matching pattern, so the full "phenotype" suffix is not used -->
+    <dynamicField name="*enotype" type="text_ws" indexed="true" stored="true" multiValued="true"/>
```

Comment:
I would add an "h" so that it is "*henotype", so that it is not (potentially) confused with "genotype" fields which we might have in the future (or have already?)